app/javascript/components/Footer/Footer.tsx

```diff
@@ -1,12 +1,77 @@
-import * as React from 'react';
+import React from 'react';
+import { Footer as UswdsFooter, GridContainer, FooterNav, Logo, Address } from '@trussworks/react-uswds';
 
+import './Footer.css';
 interface FooterProps {
 }
 
 export const Footer = (props: FooterProps) => {
+
+  const returnToTop = (
+    <GridContainer className="usa-footer__return-to-top">
+      <a href="#">Return to top</a>
+    </GridContainer>
+  )
+
+  const primaryFooterLinks = [
+    <a className="usa-footer__primary-link" href="#">
+      Primary Link 1
+    </a>,
+    <a className="usa-footer__primary-link" href="#">
+      Primary Link 2
+    </a>,
+    <a className="usa-footer__primary-link" href="#">
+      Primary Link 3
+    </a>,
+    <a className="usa-footer__primary-link" href="#">
+      Primary Link 4
+    </a>
+  ];
+
+  const addressItems = [
+    <a key="telephone" href="tel:1-800-555-5555">
+      (800) CALL-GOVT
+    </a>,
+    <a key="email" href="mailto:info@agency.gov">
+      info@agency.gov
+    </a>
+  ]
+
   return (
     <div id="serp-footer-wrapper">
-
+      <UswdsFooter
+        size="slim"
+        returnToTop={returnToTop}
+        primary={
+          <div className="usa-footer__primary-container grid-row">
+            <div className="mobile-lg:grid-col-8">
+              <FooterNav
+                size="slim"
+                links={primaryFooterLinks}
+              />
+            </div>
+            <div className="tablet:grid-col-4">
+              <Address
+                size="slim"
+                items={addressItems}
+              />
+            </div>
+          </div>
+        }
+        secondary={
```

Comment:
Instead of hiding this with CSS, we can just use `secondary={<></>}`. That way we don't need the CSS, and we can remove the `Logo` import.

Reply:
I tried it but it gives a empty gray section like this:
<img width="1435" alt="image" src="https://user-images.githubusercontent.com/6338785/218865109-c824c846-6eca-47aa-b90e-3b2ac75d940c.png">

Reply:
Ah, so we need the CSS, but we should still remove the `Logo` and use `<></>`.

Reply:
This has been updated. Kindly review.